scripts/github-actions/generate-module-index-data.js

```diff
@@ -91,6 +92,12 @@ async function generateModuleIndexData({ require, github, context, core }) {
 
     for (const moduleName of moduleNames) {
       const modulePath = `${moduleGroupPath}/${moduleName}`;
+      const archivedFilePath = `${modulePath}/ARCHIVED.md`;
+
+      if (existsSync(archivedFilePath)) {
+        continue;
+      }
+
```

Comment:
Tested the logic locally.

Reply:
Ok, so as discussed, this jumps out from the for cycle and continues with the next one, in case the ARCHIVED.md file exists - which is exactly what we need to not include those modules in the index and IntelliSense JSON that were already archived. Thanks a lot!